### DISCUSSION TITLE
Live updates?

### DISCUSSION THREAD

**Discussion by sawft99** [mcndt/obsidian-quickshare#23] [General]
If I share a note and then edit it again do I need to reshare it or is does it automatically update on its own? It doesn't look like it does but just curious. If not I don't know how much of a pain it would or if it's outside the project scope but that would be a cool feature.

**Comment by xaashley** [comment]
Hi, this is a really old question, but I figured I'd share my experience in case others are looking.

It does not appear that the share links automatically update as we make changes to the original notes.  The easiest workaround I've found for this is to simply create a new share link.  The plugin will replace any existing share links with the updated version.  No need to delete the existing link before creating an updated one.  It's not a lot of work, but would be better if the links auto-updated because sometimes I forget to refresh the link!

Anyway, hope this helps someone!
